PKGBUILD

```diff
@@ -1,7 +1,7 @@
 pkgname=obs-backgroundremoval
 pkgver=0.2.5_beta
-_ONNXRUNTIME_FILE=onnxruntime-linux-x64-1.7.0.tgz
-_onnxdir=onnxruntime-linux-x64-1.7.0
+_onnxversion=1.7.0
+_onnxdir=onnxruntime-linux-x64-${_onnxversion}
```

Comment:
you could call this "_onnxtag" and re-use for multiple purposes...

Reply:
Fine with me but I’m not sure what the procedure is – do I now change the file and add a new commit to this pull request?

Reply:
add a new commit to your master branch that it will magically appear here

Reply:
I have now done the renaming – not sure what you have in mind about reusing.

Reply:
I meant renaming the dir variable and reuse it in the source's url

Reply:
This form of communication is getting a bit too complicated for me. Can I ask you to just take the file I made and make out of it whatever you like? I really don’t mind if I’m listed as a co-author or not.